.github/workflows/tests.yml

```diff
@@ -166,11 +194,22 @@ jobs:
           sudo apt-get update
           sudo apt-get install --no-install-recommends -y libdqlite-dev pkg-config
 
+      - name: Download system test dependencies
```

Comment:
I understand here we fetch the deps from the previous build but in the step after we again run `make` to build them again. What is the reason for this?

Reply:
We probably don't need this step actually. Thanks for pointing this out!

I replicated logic from the `lxd` and `microcluster` GitHub Tiobe actions and overlooked this.

Reply:
I've cleaned up the build steps :)